packages/react/src/components/Card/__stories__/Card.stories.js

```diff
@@ -35,11 +35,11 @@ export default {
         cta: object(
```

Comment:
@annawen1 since under the hood, can this be broken out as separate knobs?

Reply:
yep done